migtests/scripts/functions.sh

```diff
@@ -1071,4 +1071,33 @@ EOF
 			exit 1
 			;;
 	esac
-}
\ No newline at end of file
+}
+
+normalize_and_export_vars() {
+    local test_suffix=$1
+
+    # Normalize TEST_NAME
+    export NORMALIZED_TEST_NAME="$(echo "$TEST_NAME" | tr '/-' '_')"
+
+    # Set EXPORT_DIR
+    export EXPORT_DIR=${EXPORT_DIR:-"${TEST_DIR}/${NORMALIZED_TEST_NAME}_${test_suffix}_export-dir"}
```

Comment:
nit: seems like test_suffix could be included in NORMALIZED_TEST_NAME? You wouldn't have to concat it multiple times like in EXPORT_DIR, SOURCE_DB_NAME, etc.

Reply:
Had to revert the change made for this since the uniqueness was lost. Added comments for now. Will add a better solution later.